packages/teleport-plugin-angular-base-component/src/utils.ts

```diff
@@ -0,0 +1,66 @@
+import * as types from '@babel/types'
+import {
+  convertValueToLiteral,
+  getTSAnnotationForType,
+} from '@teleporthq/teleport-shared/dist/cjs/utils/ast-js-utils'
+import {
+  getComponentFileName,
+  getStyleFileName,
+} from '@teleporthq/teleport-shared/dist/cjs/utils/uidl-utils'
+import { UIDLPropDefinition, UIDLStateDefinition, ComponentUIDL } from '@teleporthq/teleport-types'
+
+export const generateComponentDecorator = (uidl: ComponentUIDL, t = types) => {
+  const decoratorArgs = [
+    t.objectExpression([
+      t.objectProperty(t.identifier('selector'), t.stringLiteral('app-root')),
+      t.objectProperty(
+        t.identifier('templateUrl'),
+        t.stringLiteral(`./${getComponentFileName(uidl)}.html`)
```

Comment:
forgot about this, but please use FILE_TYPE.HTML and FILE_TYPE.css for `html` and `css` to have consistency across the entire project